packages/server/postgres/migrations/2024-10-31T14:42:16.120Z_add-use-ai-to-orgs.ts

```diff
@@ -0,0 +1,59 @@
+import {Kysely, sql} from 'kysely'
+
+export async function up(db: Kysely<any>): Promise<void> {
+  await db.schema
+    .alterTable('Organization')
+    .addColumn('useAI', 'boolean', (col) => col.notNull().defaultTo(true))
+    .execute()
+
+  await db
+    .updateTable('Organization')
+    .set({useAI: false})
+    .where(
+      'id',
+      'in',
+      db
+        .selectFrom('FeatureFlagOwner')
+        .innerJoin('FeatureFlag', 'FeatureFlag.id', 'FeatureFlagOwner.featureFlagId')
+        .where('FeatureFlag.featureName', '=', 'noAISummary')
+        .select('FeatureFlagOwner.orgId')
+    )
+    .execute()
```

Comment:
+1 Why do we have to do this? Shouldn't delete cascade for the 'FeatureFlagOwner' take care of this already?